test/createAndSendTx.spec.ts

```diff
@@ -50,7 +51,97 @@ describe("Create and Send Tx Integration Test", function () {
     await createAndSendTx(
       {
         tx: "balances.transfer",
-        params: BALTATHAR + "," + testAmount,
+        params: [BALTATHAR, testAmount],
+        address: ALITH,
+        sudo: false,
+      },
+      { ws: wsUrl, network: "moonbase" },
+      async (payload: string) => {
+        return await testSignCLIPrivateKey(payload);
+      }
+    );
+
+    // Stop producing blocks
+    produceBlocks = false;
+
+    // Then check incremented balance of Baltathar
+    const finalBalance = await getBalance(BALTATHAR, api);
+    assert.equal(
+      Number(finalBalance).toString().substring(0, 15),
+      (Number(initialBalance) + Number(testAmount)).toString().substring(0, 15)
+    );
+  });
+  it.only("should increment Baltathar's account balance - sudo -large payload", async function () {
```

Comment:
This test fails with `RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1010: Invalid Transaction: Transaction has a bad signature`

Reply:
Because params is too large